pkg/webhook/parser.go

```diff
@@ -126,6 +135,9 @@ type Config struct {
 	// /validate-batch-tutorial-kubebuilder-io-v1-cronjob
 	Path string `marker:"path,optional"`
 
+	// ServicePort indicates the port of the K8s Service the webhook uses
+	ServicePort int32 `marker:"servicePort,optional"`
```

Comment:
Should this be a pointer / can this be a pointer?

Just to make the check for unset a bit cleaner

Reply:
good idea, updated